src/components/VSelectField/VSelectField.vue

```diff
@@ -2,7 +2,7 @@
   <div
     class="relative m-0.5px box-content block w-fit overflow-hidden rounded-sm border text-sm focus-within:m-0 focus-within:border-1.5 focus-within:border-pink hover:border-dark-charcoal focus-within:hover:border-pink"
     :class="
-      variant === 'yellow' ? 'border-[#d6bb32]' : 'border-dark-charcoal-20'
+      'border-dark-charcoal border-opacity-20'
```

Comment:
Thank you for this PR, @Mayank-Jain-1! It's a great start.

We use `:class="<some JS expression>" to add dynamic classes in Vue. For example, here it was used because the class depended on the `variant` prop.

Now, all of the `VSelectField` will have the same border, so the border color classes are static, and we can add them to the static `class` list above.
You should add these two classes between `border` and `text-sm` on line 3 (there is a linter that will throw an error if the class order is incorrect).

Can you also remove the `variant` prop from this component?
https://github.com/WordPress/openverse-frontend/blob/8221e6d4dddc271427523161fdd8155407e7f4f2/src/components/VSelectField/VSelectField.vue#L103-L111

It's also used here:
https://github.com/WordPress/openverse-frontend/blob/8221e6d4dddc271427523161fdd8155407e7f4f2/src/components/VFourOhFour.vue#L47

A note on GitHub usage: Try to always create a branch for your new PRs, and not open a PR from your `main` branch. Otherwise, it will be difficult to keep your fork up-to-date, or create another PR.

Reply:
Sure i will do the changes. I am also supposed to change the ```VFourOFour.vue``` file and remove the ```:language-props="{ variant: 'yellow' }"``` from there as well?.

Reply:
I've merged your other PR, @Mayank-Jain-1, and will close this one. Could you open a new PR with the `VFourOhFour.vue` change, too?

Reply:
Sure mam, Thanks for the merge 😊